### PR TITLE
get v2 book list locally and add greek

### PR DESCRIPTION
**Pull request by jnavila**
The books'repos can now be located outside of progit2 organization.

### CODE REVIEW

File: lib/tasks/book2.rake [hunk 1]
```diff
@@ -35,18 +37,39 @@ task :remote_genbook2 => :environment do
...
+      repo_head = @octokit.ref(repo, "heads/master").object[:sha]
+      book = Book.where(:edition => 2, :code => code).first_or_create
       repo_head != book.ebook_html
     end
   end
 
-  books.each do |book|
-    repo = repo_name(book.code)
+  books.each do |code, repo|
+    book = Book.where(:edition => 2, :code => code).first_or_create
```
Comment (peff) [inline lib/tasks/book2.rake]: Can we drop the `repo_name` helper now?

**Comment by peff** [comment]
Answering my own questions with some commits: yes, we can drop `repo_name`, and yes, the xref stuff was debugging cruft and we can drop similar `appendix` lines, too.

**Comment by peff** [comment]
I also added in a link to the Greek translation from the book navbar (which for some reason exists in two near-identical files...?).

So I think it's good to go.

**Comment by jnavila** [comment]
Thanks for all the fixes. Prepare to acces more languages, Some translators have been waiting for a while.

**Pull request merged**